docs/tutorials/kubectl/app-management-overview.md

```diff
@@ -0,0 +1,238 @@
+---
+title: App Management Overview
+---
+
+{% capture overview %}
+
+*Kubectl* supports several different approaches for creating and managing
+applications in a Kubernetes cluster.  This document provides an overview
```

Comment:
Just chatted with @devin-donnelly.  WDYT of this:

- Live objects in a cluster (e.g. Deployment, Service): `Kubernetes Object` or `Object` (depending on context)
- Yaml configs declaring all or part of a Kubernetes Object: `Kubernetes Object Config` or `Object Config` (depending on context)